### DISCUSSION TITLE
Unity 接入

### DISCUSSION THREAD

**Discussion by UltraREX** [Tencent/libpag#63] [Q&A] [answered]
请问想接入 Unity 项目的话，是需要分别导出 iOS、Android 项目后，必须按照官方教程里的 iOS 和 Android 分别操作嘛，有无更好的思路或方式呢？多谢~

**Comment by domchen** [comment, marked as answer]
基本都要每个平台特殊处理的。你如果知道怎么让Unity直接用C++的，也可以写个桥接层把PAG直接封装成Unity里的组件。只有C++是跨平台的。否则就是用android和ios提供的预编译库和对应平台接口调用是最直接的。
